lib/wallaby/query.ex

```diff
@@ -54,6 +54,20 @@ defmodule Wallaby.Query do
     end
   end
   ```
+
+  ## What does my query do?
+
+  Wanna check out what exactly your query will do? Nothing easier than that with
+  `Wallaby.Query.compile/1` - it takes a query and returns you the css or xpath
```

Comment:
Instead of "Nothing easier than that with `Wallaby.Query.compile/1`", how about "Look no further than `Wallaby.Query.compile/1`"? Also maybe "it takes a query and returns ~~you~~ the css or xpath that will be sent to the driver"